hsctl

```diff
@@ -60,8 +60,9 @@ restart_hs() {
     echo "*** ${1^^} ***"
     stop_nginx
     preflight_hs
-    docker stop hydroshare_hydroshare_1
-    docker start hydroshare_hydroshare_1
+    docker-compose stop hydroshare
+    docker-compose build hydroshare
```

Comment:
@hyi - Ah yes, good catch. That line was initially a `docker-compose start hydroshare` call but changed it to `up` for the very reason that up builds and runs. Will remove the `build` call and update the PR.